layouts/universal-login/single.html

```diff
@@ -131,4 +131,39 @@ <h1 class="title">Select Your Metadata Region (Satellite)</h1>
     function setIframeSource(url) {
         document.getElementById('iframe').src = url;
     }
+
+    function selectSatelliteOnActivation(nameParam, activated) {
+        switch (nameParam) {
+            case 'Asia-East-1':
+                selectAsiaSatellite();
+                setIframeSource(`https://asia-east-1.tardigrade.io/login?activated=${activated}`);
+                break;
+            case 'Europe-West-1':
+                selectEuropeSatellite();
+                setIframeSource(`https://europe-west-1.tardigrade.io/login?activated=${activated}`);
+                break;
+            default:
+                selectUSSatellite();
+                setIframeSource(`https://us-central-1.tardigrade.io/login?activated=${activated}`);
+        }
+    }
+
+    window.onload = () => {
+        const urlParams = new URLSearchParams(window.location.search);
+        if (!urlParams.has('name')) {
+            selectUSSatellite();
+
+            return;
+        }
+
+        const nameParam = urlParams.get('name');
+
+        if (urlParams.get('activated') === 'true') {
```

Comment:
selectSatelliteOnActivation(nameParam, urlParams.get('activated') === 'true');

Reply:
this can reduce 7 rows to 1

Reply:
you're right